include/mbgl/util/run_loop.hpp

```diff
@@ -48,6 +48,15 @@ class RunLoop : public Scheduler,
     void runOnce();
     void stop();
 
+    using PlatformCallback = std::function<void()>;
```

Comment:
nit: `std::function<void()>` is itself a short and self-describing name, not sure it's worth aliasing